bin/arno-iptables-firewall

```diff
@@ -47,15 +47,17 @@ fi
 
 # Check if the environment file exists and if so, load it
 #########################################################
-if [ -z "$ENV_FILE" ]; then
-  if [ -f /usr/local/share/arno-iptables-firewall/environment ]; then
+# Autodetect according to the path of the main script
+ENV_FILE="$(dirname $0)/../environment"
+
+if [ ! -e "$ENV_FILE" ]; then
+  if [ -e /usr/local/share/arno-iptables-firewall/environment ]; then
     ENV_FILE="/usr/local/share/arno-iptables-firewall/environment"
   else
-    if [ -f /usr/share/arno-iptables-firewall/environment ]; then
+    if [ -e /usr/share/arno-iptables-firewall/environment ]; then
```

Comment:
The `-f` test is the correct choice for an regular file.

Ref: https://stackoverflow.com/questions/10204562/difference-between-if-e-and-if-f